webapp/views/App/views/Designer/TaxonomyList/HeaderLeft/HeaderLeft.js

```diff
@@ -1,26 +1,25 @@
 import React from 'react'
-import { useDispatch } from 'react-redux'
 import { appModuleUri, designerModules } from '@webapp/app/appModules'
 
 import { matchPath, useHistory, useLocation } from 'react-router'
 
-import * as TaxonomyActions from '@webapp/views/App/views/Designer/TaxonomyDetail/actions'
-
-import * as Taxonomy from '@core/survey/taxonomy'
-
 import { useI18n } from '@webapp/store/system'
+import PropTypes from 'prop-types'
 
-const HeaderLeft = () => {
+const HeaderLeft = (props) => {
   const i18n = useI18n()
-  const dispatch = useDispatch()
   const { pathname } = useLocation()
   const history = useHistory()
   const inTaxonomiesPath = Boolean(matchPath(pathname, appModuleUri(designerModules.taxonomies)))
 
+  const { headerProps } = props
+  const { onTaxonomyCreated } = headerProps
+
   const add = async () => {
-    const taxonomy = await dispatch(TaxonomyActions.createTaxonomy())
     if (inTaxonomiesPath) {
-      history.push(`${appModuleUri(designerModules.taxonomy)}${Taxonomy.getUuid(taxonomy)}/`)
+      history.push(`${appModuleUri(designerModules.taxonomy)}`)
```

Comment:
you dont need this `$

Reply:
ahahahah you know copy paste but not clean ahhaha